### ISSUE TITLE
Exporting XLSX with `date` or `time` columns fails with stacktrace

### ISSUE THREAD

**Issue by cnhongwei** [metabase/metabase#13298]
**Describe the bug**
exporting excel file can't open. 

**Logs**
{:database_id 4,
 :started_at #t "2020-09-23T14:34:36.138704+07:00[Asia/Bangkok]",
 :via
 [{:status :failed,
   :class clojure.lang.ExceptionInfo,
   :error "Error reducing result rows",
   :stacktrace
   ["--> query_processor.context.default$default_reducef$fn__37988.invoke(default.clj:61)"
    "query_processor.context.default$default_reducef.invokeStatic(default.clj:58)"
    "query_processor.context.default$default_reducef.invoke(default.clj:49)"
    "query_processor.context$reducef.invokeStatic(context.clj:69)"
    "query_processor.context$reducef.invoke(context.clj:62)"
    "query_processor.context.default$default_runf$respond_STAR___37992.invoke(default.clj:70)"
    "driver.sql_jdbc.execute$execute_reducible_query.invokeStatic(execute.clj:392)"
    "driver.sql_jdbc.execute$execute_reducible_query.invoke(execute.clj:377)"
    "driver.sql_jdbc$fn__73933.invokeStatic(sql_jdbc.clj:49)"
    "driver.sql_jdbc$fn__73933.invoke(sql_jdbc.clj:47)"
    "query_processor.context$executef.invokeStatic(context.clj:59)"
    "query_processor.context$executef.invoke(context.clj:48)"
    "query_processor.context.default$default_runf.invokeStatic(default.clj:69)"
    "query_processor.context.default$default_runf.invoke(default.clj:67)"
    "query_processor.context$runf.invokeStatic(context.clj:45)"
    "query_processor.context$runf.invoke(context.clj:39)"
    "query_processor.reducible$pivot.invokeStatic(reducible.clj:34)"
    "query_processor.reducible$pivot.invoke(reducible.clj:31)"
    "query_processor.middleware.mbql_to_native$mbql__GT_native$fn__45635.invoke(mbql_to_native.clj:26)"
    "query_processor.middleware.check_features$check_features$fn__44911.invoke(check_features.clj:42)"
    "query_processor.middleware.optimize_datetime_filters$optimize_datetime_filters$fn__45800.invoke(optimize_datetime_filters.clj:133)"
    "query_processor.middleware.wrap_value_literals$wrap_value_literals$fn__47328.invoke(wrap_value_literals.clj:137)"
    "query_processor.middleware.annotate$add_column_info$fn__43532.invoke(annotate.clj:574)"
    "query_processor.middleware.permissions$check_query_permissions$fn__44786.invoke(permissions.clj:64)"
    "query_processor.middleware.pre_alias_aggregations$pre_alias_aggregations$fn__46318.invoke(pre_alias_aggregations.clj:40)"
    "query_processor.middleware.cumulative_aggregations$handle_cumulative_aggregations$fn__44984.invoke(cumulative_aggregations.clj:61)"
    "query_processor.middleware.resolve_joins$resolve_joins$fn__46850.invoke(resolve_joins.clj:183)"
    "query_processor.middleware.add_implicit_joins$add_implicit_joins$fn__39262.invoke(add_implicit_joins.clj:245)"
    "query_processor.middleware.large_int_id$convert_id_to_string$fn__45596.invoke(large_int_id.clj:44)"
    "query_processor.middleware.limit$limit$fn__45621.invoke(limit.clj:38)"
    "query_processor.middleware.format_rows$format_rows$fn__45576.invoke(format_rows.clj:81)"
    "query_processor.middleware.desugar$desugar$fn__45050.invoke(desugar.clj:22)"
    "query_processor.middleware.binning$update_binning_strategy$fn__44076.invoke(binning.clj:229)"
    "query_processor.middleware.resolve_fields$resolve_fields$fn__44592.invoke(resolve_fields.clj:24)"
    "query_processor.middleware.add_dimension_projections$add_remapping$fn__38811.invoke(add_dimension_projections.clj:318)"
    "query_processor.middleware.add_implicit_clauses$add_implicit_clauses$fn__39018.invoke(add_implicit_clauses.clj:141)"
    "query_processor.middleware.add_source_metadata$add_source_metadata_for_source_queries$fn__39411.invoke(add_source_metadata.clj:105)"
    "query_processor.middleware.reconcile_breakout_and_order_by_bucketing$reconcile_breakout_and_order_by_bucketing$fn__46515.invoke(reconcile_breakout_and_order_by_bucketing.clj:98)"
    "query_processor.middleware.auto_bucket_datetimes$auto_bucket_datetimes$fn__43717.invoke(auto_bucket_datetimes.clj:125)"
    "query_processor.middleware.resolve_source_table$resolve_source_tables$fn__44639.invoke(resolve_source_table.clj:46)"
    "query_processor.middleware.parameters$substitute_parameters$fn__46300.invoke(parameters.clj:114)"
    "query_processor.middleware.resolve_referenced$resolve_referenced_card_resources$fn__44691.invoke(resolve_referenced.clj:80)"
    "query_processor.middleware.expand_macros$expand_macros$fn__45306.invoke(expand_macros.clj:158)"
    "query_processor.middleware.add_timezone_info$add_timezone_info$fn__39442.invoke(add_timezone_info.clj:15)"
    "query_processor.middleware.splice_params_in_response$splice_params_in_response$fn__47212.invoke(splice_params_in_response.clj:32)"
    "query_processor.middleware.resolve_database_and_driver$resolve_database_and_driver$fn__46526$fn__46530.invoke(resolve_database_and_driver.clj:33)"
    "driver$do_with_driver.invokeStatic(driver.clj:61)"
    "driver$do_with_driver.invoke(driver.clj:57)"
    "query_processor.middleware.resolve_database_and_driver$resolve_database_and_driver$fn__46526.invoke(resolve_database_and_driver.clj:27)"
    "query_processor.middleware.fetch_source_query$resolve_card_id_source_tables$fn__45524.invoke(fetch_source_query.clj:267)"
    "query_processor.middleware.store$initialize_store$fn__47221$fn__47222.invoke(store.clj:11)"
    "query_processor.store$do_with_store.invokeStatic(store.clj:46)"
    "query_processor.store$do_with_store.invoke(store.clj:40)"
    "query_processor.middleware.store$initialize_store$fn__47221.invoke(store.clj:10)"
    "query_processor.middleware.cache$maybe_return_cached_results$fn__44568.invoke(cache.clj:209)"
    "query_processor.middleware.validate$validate_query$fn__47230.invoke(validate.clj:10)"
    "query_processor.middleware.normalize_query$normalize$fn__45648.invoke(normalize_query.clj:22)"
    "query_processor.middleware.add_rows_truncated$add_rows_truncated$fn__39280.invoke(add_rows_truncated.clj:36)"
    "query_processor.middleware.results_metadata$record_and_return_metadata_BANG_$fn__47197.invoke(results_metadata.clj:143)"
    "query_processor.middleware.constraints$add_default_userland_constraints$fn__44927.invoke(constraints.clj:42)"
    "query_processor.middleware.process_userland_query$process_userland_query$fn__46389.invoke(process_userland_query.clj:136)"
    "query_processor.middleware.catch_exceptions$catch_exceptions$fn__44870.invoke(catch_exceptions.clj:174)"
    "query_processor.reducible$async_qp$qp_STAR___38074$thunk__38075.invoke(reducible.clj:101)"
    "query_processor.reducible$async_qp$qp_STAR___38074$fn__38077.invoke(reducible.clj:106)"],
   :error_type :qp,
   :ex-data {:type :qp}}],
 :error_type :qp,
 :json_query {:database 4, :query {:source-table 457}, :type "query", :middleware {:js-int-to-string? true, :skip-results-metadata? true, :format-rows? false}, :async? true},
 :native
 {:query
  "SELECT `paidpromoter`.`memberId` AS `memberId`, `paidpromoter`.`mobile` AS `mobile`, `paidpromoter`.`beginDate` AS `beginDate`, `paidpromoter`.`endDate` AS `endDate` FROM `paidpromoter` LIMIT 1048576",
  :params nil},
 :status :failed,
 :class clojure.lang.ExceptionInfo,
 :stacktrace
 ["java_time.temporal$instant.invokeStatic(temporal.clj:350)"
  "java_time.temporal$instant.invoke(temporal.clj:350)"
  "java_time.pre_java8$java_date.invokeStatic(pre_java8.clj:16)"
  "java_time.pre_java8$java_date.invoke(pre_java8.clj:8)"
  "--> query_processor.streaming.xlsx$fn__35422.invokeStatic(xlsx.clj:38)"
  "query_processor.streaming.xlsx$fn__35422.invoke(xlsx.clj:29)"
  "query_processor.streaming.xlsx$fn__35407$G__35402__35412.invoke(xlsx.clj:20)"
  "query_processor.streaming.xlsx$fn$reify__35434.write_row_BANG_(xlsx.clj:76)"
  "query_processor.streaming$streaming_rff$fn__35441$fn__35442.invoke(streaming.clj:35)"
  "query_processor.middleware.process_userland_query$add_and_save_execution_info_xform_BANG_$execution_info_rf_STAR___46380.invoke(process_userland_query.clj:95)"
  "query_processor.middleware.add_rows_truncated$add_rows_truncated_xform$fn__39275.invoke(add_rows_truncated.clj:28)"
  "query_processor.middleware.limit$limit_xform$fn__45618.invoke(limit.clj:25)"
  "query_processor.reducible$reducible_rows$reify__38090.reduce(reducible.clj:157)"
  "query_processor.context.default$default_reducef$fn__37988.invoke(default.clj:59)"
  "query_processor.context.default$default_reducef.invokeStatic(default.clj:58)"
  "query_processor.context.default$default_reducef.invoke(default.clj:49)"
  "query_processor.context$reducef.invokeStatic(context.clj:69)"
  "query_processor.context$reducef.invoke(context.clj:62)"
  "query_processor.context.default$default_runf$respond_STAR___37992.invoke(default.clj:70)"
  "driver.sql_jdbc.execute$execute_reducible_query.invokeStatic(execute.clj:392)"
  "driver.sql_jdbc.execute$execute_reducible_query.invoke(execute.clj:377)"
  "driver.sql_jdbc$fn__73933.invokeStatic(sql_jdbc.clj:49)"
  "driver.sql_jdbc$fn__73933.invoke(sql_jdbc.clj:47)"
  "query_processor.context$executef.invokeStatic(context.clj:59)"
  "query_processor.context$executef.invoke(context.clj:48)"
  "query_processor.context.default$default_runf.invokeStatic(default.clj:69)"
  "query_processor.context.default$default_runf.invoke(default.clj:67)"
  "query_processor.context$runf.invokeStatic(context.clj:45)"
  "query_processor.context$runf.invoke(context.clj:39)"
  "query_processor.reducible$pivot.invokeStatic(reducible.clj:34)"
  "query_processor.reducible$pivot.invoke(reducible.clj:31)"
  "query_processor.middleware.mbql_to_native$mbql__GT_native$fn__45635.invoke(mbql_to_native.clj:26)"
  "query_processor.middleware.check_features$check_features$fn__44911.invoke(check_features.clj:42)"
  "query_processor.middleware.optimize_datetime_filters$optimize_datetime_filters$fn__45800.invoke(optimize_datetime_filters.clj:133)"
  "query_processor.middleware.wrap_value_literals$wrap_value_literals$fn__47328.invoke(wrap_value_literals.clj:137)"
  "query_processor.middleware.annotate$add_column_info$fn__43532.invoke(annotate.clj:574)"
  "query_processor.middleware.permissions$check_query_permissions$fn__44786.invoke(permissions.clj:64)"
  "query_processor.middleware.pre_alias_aggregations$pre_alias_aggregations$fn__46318.invoke(pre_alias_aggregations.clj:40)"
  "query_processor.middleware.cumulative_aggregations$handle_cumulative_aggregations$fn__44984.invoke(cumulative_aggregations.clj:61)"
  "query_processor.middleware.resolve_joins$resolve_joins$fn__46850.invoke(resolve_joins.clj:183)"
  "query_processor.middleware.add_implicit_joins$add_implicit_joins$fn__39262.invoke(add_implicit_joins.clj:245)"
  "query_processor.middleware.large_int_id$convert_id_to_string$fn__45596.invoke(large_int_id.clj:44)"
  "query_processor.middleware.limit$limit$fn__45621.invoke(limit.clj:38)"
  "query_processor.middleware.format_rows$format_rows$fn__45576.invoke(format_rows.clj:81)"
  "query_processor.middleware.desugar$desugar$fn__45050.invoke(desugar.clj:22)"
  "query_processor.middleware.binning$update_binning_strategy$fn__44076.invoke(binning.clj:229)"
  "query_processor.middleware.resolve_fields$resolve_fields$fn__44592.invoke(resolve_fields.clj:24)"
  "query_processor.middleware.add_dimension_projections$add_remapping$fn__38811.invoke(add_dimension_projections.clj:318)"
  "query_processor.middleware.add_implicit_clauses$add_implicit_clauses$fn__39018.invoke(add_implicit_clauses.clj:141)"
  "query_processor.middleware.add_source_metadata$add_source_metadata_for_source_queries$fn__39411.invoke(add_source_metadata.clj:105)"
  "query_processor.middleware.reconcile_breakout_and_order_by_bucketing$reconcile_breakout_and_order_by_bucketing$fn__46515.invoke(reconcile_breakout_and_order_by_bucketing.clj:98)"
  "query_processor.middleware.auto_bucket_datetimes$auto_bucket_datetimes$fn__43717.invoke(auto_bucket_datetimes.clj:125)"
  "query_processor.middleware.resolve_source_table$resolve_source_tables$fn__44639.invoke(resolve_source_table.clj:46)"
  "query_processor.middleware.parameters$substitute_parameters$fn__46300.invoke(parameters.clj:114)"
  "query_processor.middleware.resolve_referenced$resolve_referenced_card_resources$fn__44691.invoke(resolve_referenced.clj:80)"
  "query_processor.middleware.expand_macros$expand_macros$fn__45306.invoke(expand_macros.clj:158)"
  "query_processor.middleware.add_timezone_info$add_timezone_info$fn__39442.invoke(add_timezone_info.clj:15)"
  "query_processor.middleware.splice_params_in_response$splice_params_in_response$fn__47212.invoke(splice_params_in_response.clj:32)"
  "query_processor.middleware.resolve_database_and_driver$resolve_database_and_driver$fn__46526$fn__46530.invoke(resolve_database_and_driver.clj:33)"
  "driver$do_with_driver.invokeStatic(driver.clj:61)"
  "driver$do_with_driver.invoke(driver.clj:57)"
  "query_processor.middleware.resolve_database_and_driver$resolve_database_and_driver$fn__46526.invoke(resolve_database_and_driver.clj:27)"
  "query_processor.middleware.fetch_source_query$resolve_card_id_source_tables$fn__45524.invoke(fetch_source_query.clj:267)"
  "query_processor.middleware.store$initialize_store$fn__47221$fn__47222.invoke(store.clj:11)"
  "query_processor.store$do_with_store.invokeStatic(store.clj:46)"
  "query_processor.store$do_with_store.invoke(store.clj:40)"
  "query_processor.middleware.store$initialize_store$fn__47221.invoke(store.clj:10)"
  "query_processor.middleware.cache$maybe_return_cached_results$fn__44568.invoke(cache.clj:209)"
  "query_processor.middleware.validate$validate_query$fn__47230.invoke(validate.clj:10)"
  "query_processor.middleware.normalize_query$normalize$fn__45648.invoke(normalize_query.clj:22)"
  "query_processor.middleware.add_rows_truncated$add_rows_truncated$fn__39280.invoke(add_rows_truncated.clj:36)"
  "query_processor.middleware.results_metadata$record_and_return_metadata_BANG_$fn__47197.invoke(results_metadata.clj:143)"
  "query_processor.middleware.constraints$add_default_userland_constraints$fn__44927.invoke(constraints.clj:42)"
  "query_processor.middleware.process_userland_query$process_userland_query$fn__46389.invoke(process_userland_query.clj:136)"
  "query_processor.middleware.catch_exceptions$catch_exceptions$fn__44870.invoke(catch_exceptions.clj:174)"
  "query_processor.reducible$async_qp$qp_STAR___38074$thunk__38075.invoke(reducible.clj:101)"
  "query_processor.reducible$async_qp$qp_STAR___38074$fn__38077.invoke(reducible.clj:106)"],
 :context :xlsx-download,
 :error "Could not convert [#t \"2020-06-01\"] to class java.time.Instant!",
 :row_count 0,
 :running_time 0,
 :preprocessed
 {:database 4,
  :query {:source-table 457, :fields [[:field-id 7336] [:field-id 7335] [:datetime-field [:field-id 7337] :default] [:datetime-field [:field-id 7338] :default]], :limit 1048576},
  :type :query,
  :middleware {:js-int-to-string? true, :skip-results-metadata? true, :format-rows? false},
  :async? true,
  :info
  {:executed-by 1,
   :context :xlsx-download,
   :query-hash [40, -36, 44, 50, 119, 105, -55, 12, 122, 51, -57, 6, -20, 100, 91, -99, 78, 26, -107, 7, 9, -38, -87, -58, 32, -72, -19, -13, -28, -11, -8, -99]}},
 :ex-data {:arguments [#t "2020-06-01"], :constructor #object[java_time.temporal$instant 0x1f18ffa0 "java_time.temporal$instant@1f18ffa0"], :to java.time.Instant},
 :data {:rows [], :cols []}}

**To Reproduce**
1. Go to 'Browser Data'
2. Open Table
3. Click "Download full results to .xlsx"
4. Open download file fail.
5. Click "Download full results to .csv" correct

**Expected behavior**
download file correct excel file

**Screenshots**

**Information about your Metabase Installation:**

You can get this information by going to Admin -> Troubleshooting.

- Your browser and the version: (Firefox 80.0,)
- Your operating system: (OS X 10.15.6)
- Your databases: (MySQL 5.7.31)
- Metabase version: (0.36.6)
- Metabase hosting environment: (Docker on Centos 7)
- Metabase internal database: (MySQL 5.7.31)

**Severity**
blocking my usage of Metabase, i just download full results to .csv instead.

**Comment by flamber** [comment]
Steps to reproduce:
1. Simple question > Sample Dataset > People
2. Export as XLSX - the downloaded file is a stacktrace
```
{"database_id":4,"started_at":"2020-09-23T11:19:34.112882+02:00","via":[{"status":"failed","class":"class clojure.lang.ExceptionInfo","error":"Error reducing result rows","stacktrace":["--> query_processor.context.default$default_reducef$fn__37988.invoke(default.clj:61)","query_processor.context.default$default_reducef.invokeStatic(default.clj:58)","query_processor.context.default$default_reducef.invoke(default.clj:49)","query_processor.context$reducef.invokeStatic(context.clj:69)","query_processor.context$reducef.invoke(context.clj:62)","query_processor.context.default$default_runf$respond_STAR___37992.invoke(default.clj:70)","driver.sql_jdbc.execute$execute_reducible_query.invokeStatic(execute.clj:392)","driver.sql_jdbc.execute$execute_reducible_query.invoke(execute.clj:377)","driver.sql_jdbc$fn__73933.invokeStatic(sql_jdbc.clj:49)","driver.sql_jdbc$fn__73933.invoke(sql_jdbc.clj:47)","driver.h2$fn__72752.invokeStatic(h2.clj:82)","driver.h2$fn__72752.invoke(h2.clj:79)","query_processor.context$executef.invokeStatic(context.clj:59)","query_processor.context$executef.invoke(context.clj:48)","query_processor.context.default$default_runf.invokeStatic(default.clj:69)","query_processor.context.default$default_runf.invoke(default.clj:67)","query_processor.context$runf.invokeStatic(context.clj:45)","query_processor.context$runf.invoke(context.clj:39)","query_processor.reducible$pivot.invokeStatic(reducible.clj:34)","query_processor.reducible$pivot.invoke(reducible.clj:31)","query_processor.middleware.mbql_to_native$mbql__GT_native$fn__45635.invoke(mbql_to_native.clj:26)","query_processor.middleware.check_features$check_features$fn__44911.invoke(check_features.clj:42)","query_processor.middleware.optimize_datetime_filters$optimize_datetime_filters$fn__45800.invoke(optimize_datetime_filters.clj:133)","query_processor.middleware.wrap_value_literals$wrap_value_literals$fn__47328.invoke(wrap_value_literals.clj:137)","query_processor.middleware.annotate$add_column_info$fn__43532.invoke(annotate.clj:574)","query_processor.middleware.permissions$check_query_permissions$fn__44786.invoke(permissions.clj:64)","query_processor.middleware.pre_alias_aggregations$pre_alias_aggregations$fn__46318.invoke(pre_alias_aggregations.clj:40)","query_processor.middleware.cumulative_aggregations$handle_cumulative_aggregations$fn__44984.invoke(cumulative_aggregations.clj:61)","query_processor.middleware.resolve_joins$resolve_joins$fn__46850.invoke(resolve_joins.clj:183)","query_processor.middleware.add_implicit_joins$add_implicit_joins$fn__39262.invoke(add_implicit_joins.clj:245)","query_processor.middleware.large_int_id$convert_id_to_string$fn__45596.invoke(large_int_id.clj:44)","query_processor.middleware.limit$limit$fn__45621.invoke(limit.clj:38)","query_processor.middleware.format_rows$format_rows$fn__45576.invoke(format_rows.clj:81)","query_processor.middleware.desugar$desugar$fn__45050.invoke(desugar.clj:22)","query_processor.middleware.binning$update_binning_strategy$fn__44076.invoke(binning.clj:229)","query_processor.middleware.resolve_fields$resolve_fields$fn__44592.invoke(resolve_fields.clj:24)","query_processor.middleware.add_dimension_projections$add_remapping$fn__38811.invoke(add_dimension_projections.clj:318)","query_processor.middleware.add_implicit_clauses$add_implicit_clauses$fn__39018.invoke(add_implicit_clauses.clj:141)","query_processor.middleware.add_source_metadata$add_source_metadata_for_source_queries$fn__39411.invoke(add_source_metadata.clj:105)","query_processor.middleware.reconcile_breakout_and_order_by_bucketing$reconcile_breakout_and_order_by_bucketing$fn__46515.invoke(reconcile_breakout_and_order_by_bucketing.clj:98)","query_processor.middleware.auto_bucket_datetimes$auto_bucket_datetimes$fn__43717.invoke(auto_bucket_datetimes.clj:125)","query_processor.middleware.resolve_source_table$resolve_source_tables$fn__44639.invoke(resolve_source_table.clj:46)","query_processor.middleware.parameters$substitute_parameters$fn__46300.invoke(parameters.clj:114)","query_processor.middleware.resolve_referenced$resolve_referenced_card_resources$fn__44691.invoke(resolve_referenced.clj:80)","query_processor.middleware.expand_macros$expand_macros$fn__45306.invoke(expand_macros.clj:158)","query_processor.middleware.add_timezone_info$add_timezone_info$fn__39442.invoke(add_timezone_info.clj:15)","query_processor.middleware.splice_params_in_response$splice_params_in_response$fn__47212.invoke(splice_params_in_response.clj:32)","query_processor.middleware.resolve_database_and_driver$resolve_database_and_driver$fn__46526$fn__46530.invoke(resolve_database_and_driver.clj:33)","driver$do_with_driver.invokeStatic(driver.clj:61)","driver$do_with_driver.invoke(driver.clj:57)","query_processor.middleware.resolve_database_and_driver$resolve_database_and_driver$fn__46526.invoke(resolve_database_and_driver.clj:27)","query_processor.middleware.fetch_source_query$resolve_card_id_source_tables$fn__45524.invoke(fetch_source_query.clj:267)","query_processor.middleware.store$initialize_store$fn__47221$fn__47222.invoke(store.clj:11)","query_processor.store$do_with_store.invokeStatic(store.clj:46)","query_processor.store$do_with_store.invoke(store.clj:40)","query_processor.middleware.store$initialize_store$fn__47221.invoke(store.clj:10)","query_processor.middleware.cache$maybe_return_cached_results$fn__44568.invoke(cache.clj:209)","query_processor.middleware.validate$validate_query$fn__47230.invoke(validate.clj:10)","query_processor.middleware.normalize_query$normalize$fn__45648.invoke(normalize_query.clj:22)","query_processor.middleware.add_rows_truncated$add_rows_truncated$fn__39280.invoke(add_rows_truncated.clj:36)","query_processor.middleware.results_metadata$record_and_return_metadata_BANG_$fn__47197.invoke(results_metadata.clj:143)","query_processor.middleware.constraints$add_default_userland_constraints$fn__44927.invoke(constraints.clj:42)","query_processor.middleware.process_userland_query$process_userland_query$fn__46389.invoke(process_userland_query.clj:136)","query_processor.middleware.catch_exceptions$catch_exceptions$fn__44870.invoke(catch_exceptions.clj:174)","query_processor.reducible$async_qp$qp_STAR___38074$thunk__38075.invoke(reducible.clj:101)","query_processor.reducible$async_qp$qp_STAR___38074$fn__38077.invoke(reducible.clj:106)"],"error_type":"qp","ex-data":{"type":"qp"}}],"error_type":"qp","json_query":{"database":4,"query":{"source-table":12},"type":"query","middleware":{"js-int-to-string?":true,"skip-results-metadata?":true,"format-rows?":false},"async?":true},"native":{"query":"SELECT \"PUBLIC\".\"PEOPLE\".\"ID\" AS \"ID\", \"PUBLIC\".\"PEOPLE\".\"ADDRESS\" AS \"ADDRESS\", \"PUBLIC\".\"PEOPLE\".\"EMAIL\" AS \"EMAIL\", \"PUBLIC\".\"PEOPLE\".\"PASSWORD\" AS \"PASSWORD\", \"PUBLIC\".\"PEOPLE\".\"NAME\" AS \"NAME\", \"PUBLIC\".\"PEOPLE\".\"CITY\" AS \"CITY\", \"PUBLIC\".\"PEOPLE\".\"LONGITUDE\" AS \"LONGITUDE\", \"PUBLIC\".\"PEOPLE\".\"STATE\" AS \"STATE\", \"PUBLIC\".\"PEOPLE\".\"SOURCE\" AS \"SOURCE\", \"PUBLIC\".\"PEOPLE\".\"BIRTH_DATE\" AS \"BIRTH_DATE\", \"PUBLIC\".\"PEOPLE\".\"ZIP\" AS \"ZIP\", \"PUBLIC\".\"PEOPLE\".\"LATITUDE\" AS \"LATITUDE\", \"PUBLIC\".\"PEOPLE\".\"CREATED_AT\" AS \"CREATED_AT\" FROM \"PUBLIC\".\"PEOPLE\" LIMIT 1048576","params":null},"status":"failed","class":"class clojure.lang.ExceptionInfo","stacktrace":["java_time.temporal$instant.invokeStatic(temporal.clj:350)","java_time.temporal$instant.invoke(temporal.clj:350)","java_time.pre_java8$java_date.invokeStatic(pre_java8.clj:16)","java_time.pre_java8$java_date.invoke(pre_java8.clj:8)","--> query_processor.streaming.xlsx$fn__35422.invokeStatic(xlsx.clj:38)","query_processor.streaming.xlsx$fn__35422.invoke(xlsx.clj:29)","query_processor.streaming.xlsx$fn__35407$G__35402__35412.invoke(xlsx.clj:20)","query_processor.streaming.xlsx$fn$reify__35434.write_row_BANG_(xlsx.clj:76)","query_processor.streaming$streaming_rff$fn__35441$fn__35442.invoke(streaming.clj:35)","query_processor.middleware.process_userland_query$add_and_save_execution_info_xform_BANG_$execution_info_rf_STAR___46380.invoke(process_userland_query.clj:95)","query_processor.middleware.add_rows_truncated$add_rows_truncated_xform$fn__39275.invoke(add_rows_truncated.clj:28)","query_processor.middleware.limit$limit_xform$fn__45618.invoke(limit.clj:25)","query_processor.middleware.large_int_id$result_int__GT_string$fn__45589.invoke(large_int_id.clj:12)","query_processor.reducible$reducible_rows$reify__38090.reduce(reducible.clj:157)","query_processor.context.default$default_reducef$fn__37988.invoke(default.clj:59)","query_processor.context.default$default_reducef.invokeStatic(default.clj:58)","query_processor.context.default$default_reducef.invoke(default.clj:49)","query_processor.context$reducef.invokeStatic(context.clj:69)","query_processor.context$reducef.invoke(context.clj:62)","query_processor.context.default$default_runf$respond_STAR___37992.invoke(default.clj:70)","driver.sql_jdbc.execute$execute_reducible_query.invokeStatic(execute.clj:392)","driver.sql_jdbc.execute$execute_reducible_query.invoke(execute.clj:377)","driver.sql_jdbc$fn__73933.invokeStatic(sql_jdbc.clj:49)","driver.sql_jdbc$fn__73933.invoke(sql_jdbc.clj:47)","driver.h2$fn__72752.invokeStatic(h2.clj:82)","driver.h2$fn__72752.invoke(h2.clj:79)","query_processor.context$executef.invokeStatic(context.clj:59)","query_processor.context$executef.invoke(context.clj:48)","query_processor.context.default$default_runf.invokeStatic(default.clj:69)","query_processor.context.default$default_runf.invoke(default.clj:67)","query_processor.context$runf.invokeStatic(context.clj:45)","query_processor.context$runf.invoke(context.clj:39)","query_processor.reducible$pivot.invokeStatic(reducible.clj:34)","query_processor.reducible$pivot.invoke(reducible.clj:31)","query_processor.middleware.mbql_to_native$mbql__GT_native$fn__45635.invoke(mbql_to_native.clj:26)","query_processor.middleware.check_features$check_features$fn__44911.invoke(check_features.clj:42)","query_processor.middleware.optimize_datetime_filters$optimize_datetime_filters$fn__45800.invoke(optimize_datetime_filters.clj:133)","query_processor.middleware.wrap_value_literals$wrap_value_literals$fn__47328.invoke(wrap_value_literals.clj:137)","query_processor.middleware.annotate$add_column_info$fn__43532.invoke(annotate.clj:574)","query_processor.middleware.permissions$check_query_permissions$fn__44786.invoke(permissions.clj:64)","query_processor.middleware.pre_alias_aggregations$pre_alias_aggregations$fn__46318.invoke(pre_alias_aggregations.clj:40)","query_processor.middleware.cumulative_aggregations$handle_cumulative_aggregations$fn__44984.invoke(cumulative_aggregations.clj:61)","query_processor.middleware.resolve_joins$resolve_joins$fn__46850.invoke(resolve_joins.clj:183)","query_processor.middleware.add_implicit_joins$add_implicit_joins$fn__39262.invoke(add_implicit_joins.clj:245)","query_processor.middleware.large_int_id$convert_id_to_string$fn__45596.invoke(large_int_id.clj:44)","query_processor.middleware.limit$limit$fn__45621.invoke(limit.clj:38)","query_processor.middleware.format_rows$format_rows$fn__45576.invoke(format_rows.clj:81)","query_processor.middleware.desugar$desugar$fn__45050.invoke(desugar.clj:22)","query_processor.middleware.binning$update_binning_strategy$fn__44076.invoke(binning.clj:229)","query_processor.middleware.resolve_fields$resolve_fields$fn__44592.invoke(resolve_fields.clj:24)","query_processor.middleware.add_dimension_projections$add_remapping$fn__38811.invoke(add_dimension_projections.clj:318)","query_processor.middleware.add_implicit_clauses$add_implicit_clauses$fn__39018.invoke(add_implicit_clauses.clj:141)","query_processor.middleware.add_source_metadata$add_source_metadata_for_source_queries$fn__39411.invoke(add_source_metadata.clj:105)","query_processor.middleware.reconcile_breakout_and_order_by_bucketing$reconcile_breakout_and_order_by_bucketing$fn__46515.invoke(reconcile_breakout_and_order_by_bucketing.clj:98)","query_processor.middleware.auto_bucket_datetimes$auto_bucket_datetimes$fn__43717.invoke(auto_bucket_datetimes.clj:125)","query_processor.middleware.resolve_source_table$resolve_source_tables$fn__44639.invoke(resolve_source_table.clj:46)","query_processor.middleware.parameters$substitute_parameters$fn__46300.invoke(parameters.clj:114)","query_processor.middleware.resolve_referenced$resolve_referenced_card_resources$fn__44691.invoke(resolve_referenced.clj:80)","query_processor.middleware.expand_macros$expand_macros$fn__45306.invoke(expand_macros.clj:158)","query_processor.middleware.add_timezone_info$add_timezone_info$fn__39442.invoke(add_timezone_info.clj:15)","query_processor.middleware.splice_params_in_response$splice_params_in_response$fn__47212.invoke(splice_params_in_response.clj:32)","query_processor.middleware.resolve_database_and_driver$resolve_database_and_driver$fn__46526$fn__46530.invoke(resolve_database_and_driver.clj:33)","driver$do_with_driver.invokeStatic(driver.clj:61)","driver$do_with_driver.invoke(driver.clj:57)","query_processor.middleware.resolve_database_and_driver$resolve_database_and_driver$fn__46526.invoke(resolve_database_and_driver.clj:27)","query_processor.middleware.fetch_source_query$resolve_card_id_source_tables$fn__45524.invoke(fetch_source_query.clj:267)","query_processor.middleware.store$initialize_store$fn__47221$fn__47222.invoke(store.clj:11)","query_processor.store$do_with_store.invokeStatic(store.clj:46)","query_processor.store$do_with_store.invoke(store.clj:40)","query_processor.middleware.store$initialize_store$fn__47221.invoke(store.clj:10)","query_processor.middleware.cache$maybe_return_cached_results$fn__44568.invoke(cache.clj:209)","query_processor.middleware.validate$validate_query$fn__47230.invoke(validate.clj:10)","query_processor.middleware.normalize_query$normalize$fn__45648.invoke(normalize_query.clj:22)","query_processor.middleware.add_rows_truncated$add_rows_truncated$fn__39280.invoke(add_rows_truncated.clj:36)","query_processor.middleware.results_metadata$record_and_return_metadata_BANG_$fn__47197.invoke(results_metadata.clj:143)","query_processor.middleware.constraints$add_default_userland_constraints$fn__44927.invoke(constraints.clj:42)","query_processor.middleware.process_userland_query$process_userland_query$fn__46389.invoke(process_userland_query.clj:136)","query_processor.middleware.catch_exceptions$catch_exceptions$fn__44870.invoke(catch_exceptions.clj:174)","query_processor.reducible$async_qp$qp_STAR___38074$thunk__38075.invoke(reducible.clj:101)","query_processor.reducible$async_qp$qp_STAR___38074$fn__38077.invoke(reducible.clj:106)"],"context":"xlsx-download","error":"Could not convert [#t \"1986-12-12\"] to class java.time.Instant!","row_count":0,"running_time":0,"preprocessed":{"database":4,"query":{"source-table":12,"fields":[["field-id",92],["field-id",93],["field-id",96],["field-id",100],["field-id",91],["field-id",101],["field-id",94],["field-id",99],["field-id",95],["datetime-field",["field-id",90],"default"],["field-id",98],["field-id",89],["datetime-field",["field-id",97],"default"]],"limit":1048576},"type":"query","middleware":{"js-int-to-string?":true,"skip-results-metadata?":true,"format-rows?":false},"async?":true,"info":{"executed-by":1,"context":"xlsx-download","query-hash":"0xB90A1A02"}},"ex-data":{"arguments":["1986-12-12"],"constructor":"java_time.temporal$instant@7ce5c4a1","to":"class java.time.Instant"},"data":{"rows":[],"cols":[]}}
```

**Comment by flamber** [comment]
Fixed by #13447
